packages/@aws-cdk/aws-apprunner/lib/service.ts

```diff
@@ -0,0 +1,603 @@
+import * as ecr from '@aws-cdk/aws-ecr';
+import { DockerImageAsset } from '@aws-cdk/aws-ecr-assets';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnService } from './';
+
+/**
+ * The image repository types
+ */
+export enum ImageRepositoryType {
+  /**
+   * Amazon ECR Public
+   */
+  ECR_PUBLIC = 'ECR_PUBLIC',
+  /**
+   * Amazon ECR
+   */
+  ECR = 'ECR',
+}
+
+/**
+ * The number of CPU units reserved for each instance of your App Runner service.
+ */
+export enum Cpu {
+  /**
+   * 1 vCPU
+   */
+  ONE_VCPU = '1 vCPU',
+  /**
+   * 2 vCPU
+   */
+  TWO_VCPU = '2 vCPU',
+}
+
+/**
+ * The amount of memory reserved for each instance of your App Runner service.
+ */
+export enum Memory {
+  /**
+   * 2 GB
+   */
+  TWO_GB = '2 GB',
+  /**
+   * 3 GB
+   */
+  THREE_GB = '3 GB',
+  /**
+   * 4 GB
+   */
+  FOUR_GB = '4 GB',
+}
+
+/**
+ * The code runtimes
+ */
+export enum CodeRuntime {
+  /**
+   * NodeJS 12
+   */
+  NODEJS_12 = 'NODEJS_12',
+  /**
+   * Python 3
+   */
+  PYTHON_3 = 'PYTHON_3',
+}
+
+/**
+ * Describes the configuration that AWS App Runner uses to run an App Runner service
+ * using an image pulled from a source image repository.
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apprunner-service-imageconfiguration.html
+ */
+interface ImageConfiguration {
+  /**
+   * The port that your application listens to in the container.
+   *
+   * @default 8080
+   */
+  readonly port?: string;
+  /**
+   * Environment variables that are available to your running App Runner service.
+   *
+   * @default - no environment variables
+   */
+  readonly environments?: { [key: string]: string };
```

Comment:
For consistency with other places with environment variables:

```suggestion
  readonly environment?: { [key: string]: string };
```